reactivesearch/src/index.js

```diff
@@ -1,7 +1,117 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import  { createRoot } from 'react-dom/client';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+//import OpenLogClient from './js/OpenLogClient'
+var OpenLogClient = require('./ts/OpenLogClient.ts').OpenLogClient;
+
+const {
+	CollectorModule,
+	Context,
+	cookieSessionResolver,
+	debounce,
+	DefaultWriter,
+	FiredSearchCollector,
+	InstantSearchQueryCollector,
+	positionResolver,
+	Query,
+	Sentinel,
+	Trail,
+	TrailType,
+	DebugWriter,
+	QueryWriter,
+	TrailWriter,
+	JSONEnvelopeWriter,
+	RedirectCollector,
+	BrowserTrackingWriter,
+	ProductClickCollector,
+	ImpressionCollector,
+	SearchResultCollector,
+	BasketClickCollector,
+	CheckoutClickCollector,
+	ConsoleTransport,
+	SuggestSearchCollector,
+	AssociatedProductCollector,
+	ListenerType,
+} = window.SearchCollector;
+
+
+/**
+ * Used for demo purposes only, store all events in localstorage.
+ * For live environments rely on DefaultWriter class shipped with the search-collector packages
+ */
+class DemoWriter {
```

Comment:
should DemoWriter, if we want to keep it, be in it's own file?

Reply:
That should be removed.  the OLWriter replaces it.